client/src/main/scala/com.gu.contentapi.client/model/Queries.scala

```diff
@@ -174,16 +194,6 @@ case class AtomUsageQuery(atomType: AtomType, atomId: String, parameterHolder: M
   override def pathSegment: String = s"atom/${atomType.toString.toLowerCase}/$atomId/usage"
 }
 
-case class RecipesQuery(parameterHolder: Map[String, Parameter] = Map.empty)
```

Comment:
Pulling this could cause clients built using it to break. Might be wise to deprecate first?

Reply:
My understanding is that there aren't any because we never published anything beyond a token recipe atom (and that is the effective alias for this query)

Reply:
I can't see any uses of this in current MAPI or dotcom repos, so it's probably a non-issue. But better to do it nicely just in case there's something lurking somewhere 👍